docs/ingestion/compaction.md

```diff
@@ -118,14 +118,14 @@ To perform a manual compaction, you submit a compaction task. Compaction tasks m
 
 |Field|Description|Required|
 |-----|-----------|--------|
-|`type`|Task type. Should be `compact`|Yes|
-|`id`|Task id|No|
-|`dataSource`|Data source name to compact|Yes|
+|`type`|Task type. Set the value to `compact`.|Yes|
+|`id`|Task ID.|No|
+|`dataSource`|Data source name to compact.|Yes|
 |`ioConfig`|I/O configuration for compaction task. See [Compaction I/O configuration](#compaction-io-configuration) for details.|Yes|
 |`dimensionsSpec`|Custom `dimensionsSpec`. The compaction task uses the specified `dimensionsSpec` if it exists instead of generating one. See [Compaction dimensionsSpec](#compaction-dimensions-spec) for details.|No|
 |`transformSpec`|Custom `transformSpec`. The compaction task uses the specified `transformSpec` rather than using `null`. See [Compaction transformSpec](#compaction-transform-spec) for details.|No|
 |`metricsSpec`|Custom `metricsSpec`. The compaction task uses the specified `metricsSpec` rather than generating one.|No|
-|`segmentGranularity`|When set, the compaction task changes the segment granularity for the given interval.  Deprecated. Use `granularitySpec`. |No|
+|`segmentGranularity`|When set, the compaction task changes the segment granularity for the given interval. Deprecated. Use `granularitySpec`. |No|
```

Comment:
Suggestion:

> If null (default), the new segments have the same segment granularity as the old segments. Set to specify a different segment granularity. Deprecated. Use `granularitySpec`.

Or, since it is deprecated, omit the explanation:

> Deprecated. Use `granularitySpec`.